bloggy/urls.py

```diff
@@ -1,63 +1,13 @@
-"""bloggy URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/2.1/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
-from allauth.account.views import (
-    LoginView,
-    LogoutView,
-    PasswordResetDoneView,
-    PasswordResetFromKeyDoneView,
-    PasswordResetFromKeyView,
-    PasswordResetView,
-    SignupView,
-)
-from django.conf.urls import url
 from django.contrib import admin
-from django.contrib.auth.decorators import user_passes_test
-from django.urls import include, path
+from django.urls import path, include
 
 from .api import router
 
-logged_users_redirect = user_passes_test(lambda u: u.is_anonymous, "/")
-
 urlpatterns = [
+    path("admin/", admin.site.urls),
+    path('accounts/', include('allauth.urls')),
```

Comment:
Well this is debatable. 
1. Custom URLs allow for redirection of logged users
2. Custom login views allow for case insensitive authentication
3. IIRC I didn't like the way `allauth` handled email management. It allowed to have many emails and I wasn't fond of this idea.

Reply:
1- django allauth do the same way
2- but it not work yet
3- i do that because i using ajax with django allauth

Reply:
1. Checked it and you're right. I don't think it used to be that way a year ago.
2. Why? Can you provide a negative case?
3. Ok but it would be nice to either disable or add templates for URLs
```
/accounts/inactive/ # dunno what this endpoint even does
/accounts/password/change/
/accounts/password/set/
/accounts/email/ # I don't want it to allow multiple emails
/accounts/social/ # This wouldn't be used for now, unless you plan to add integrations for social sites
```

Reply:
ok i will do that